workshops/models.py

```diff
@@ -228,9 +228,15 @@ def get_by_ident(ident):
         The real indicator is the 'published' flag.
         '''
 
+        # match event ID
         if re.match(r'^\d+$', ident):
             return Event.objects.get(id=ident)
 
+        # match event slug in format YYYY-MM-****
+        if re.match(r'^\d{4}-\d{2}-.+$', ident):
```

Comment:
Welp, you're right, it does capture these cases, but there's no error because all we do in both cases is `return Event.objects.get(slug=ident)`. I'll add fixing commit in a few minutes.